packages/trezor-user-env-link/package.json

```diff
@@ -11,5 +11,8 @@
     "dependencies": {
         "@trezor/utils": "workspace:*",
         "ws": "^8.16.0"
+    },
+    "devDependencies": {
+        "@types/ws": "^8.5.5"
```

Comment:
Is this change related to this PR? :thinking:

Reply:
Yes, this types wasn't installed because of `yarn focus` so they need to be explicitelly added.